src/acquisition/covidcast/database.py

```diff
@@ -240,7 +240,7 @@ def insert_or_update_batch(self, cc_rows, batch_size=2**20, commit_partial=False
   def get_covidcast_meta(self):
     """Compute and return metadata on all non-WIP COVIDcast signals."""
 
-    n_threads = cpu_count() - 1 # aka number of concurrent db connections, which [sh|c]ould be ~<= #cores available to SQL server
+    n_threads = cpu_count()*9//10 # aka number of concurrent db connections, which [sh|c]ould be ~<= 90% of the #cores available to SQL server
```

Comment:
this breaks on single core machines, but do they even make those any more??  you could do a `max(1, cpu_count()*9//10`) just in case

Reply:
gooood point ```suggestion
    n_threads = max(1, cpu_count()*9//10) # aka number of concurrent db connections, which [sh|c]ould be ~<= 90% of the #cores available to SQL server
```